### PR TITLE
docs: adds information_schema content

### LINKED ISSUE #869: Update docs for fix: columns table in information_schema misses some columns
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/3639

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

Adds the content of `information_schema`.

close #869 

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.7/en/reference/sql/information-schema/tables.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The description of columns in the `TABLES` table is as follows:
+
+- `table_catalog`: The catalog to which the table belongs. The value is always `greptime`.
+- `table_schema`: The schema to which the table belongs.
+- `table_name`: The name of the table.
+- `table_type`: The type of the table.
+  - `BASE TABLE`
+  - `TEMPORARY`
+  - `VIEW`
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/tables.md:61]: Can you describe the three types of table?

File: docs/v0.7/en/reference/sql/information-schema/table-constraints.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# TABLE_CONSTRAINTS
+todo
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/table-constraints.md:2]: we do not have a table named `TABLE_CONSTRAINTS`.

File: docs/v0.7/en/reference/sql/information-schema/columns.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+column_default: current_timestamp()
+   is_nullable: No
+   column_type: TimestampMillisecond
+column_comment: NULL
+3 rows in set (0.01 sec)
+```
+
+The description of columns in the `COLUMNS` table is as follows:
+
+- `table_catalog`: The name of the catalog to which the table with the column belongs. The value is always `greptime`.
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/columns.md:75]: I'm not sure if the value is always `greptime`.

File: docs/v0.7/en/reference/sql/information-schema/greptime-region-peers.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# GREPTIME_REGION_PEERS
+
+The `GREPTIME_REGION_PEERS` table shows detailed information of a single Region node in GreptimeDB, such as whether it is a learner or leader.
+
+:::tip NOTE
+This table is not available on [GreptimeCloud](https://greptime.cloud/).
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/greptime-region-peers.md:6]: @killme2008 please confirm it

File: docs/v0.7/en/reference/sql/information-schema/key-column-usage.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+- `constraint_catalog`: The name of the catalog to which the constraint belongs. The value is always `def`.
+- `constraint_schema`: The name of the schema to which the constraint belongs.
+- `constraint_name`: The name of the constraint.
+- `table_catalog`: The name of the catalog to which the table with the constraint belongs. The value is always `def`.
+- `real_table_catalog`: The real name of the catalog to which the table with the constraint belongs. The value is always `greptime`.
+- `table_schema`: The name of the schema to which the table belongs.
+- `table_name`: The name of the table with the constraint.
+- `column_name`: The name of the column with the constraint.
+- `ordinal_position`: The position of the column in the constraint, rather than in the table. The position number starts from `1`.
+- `position_in_unique_constraint`: The unique constraint and the primary key constraint are empty. For foreign key constraints, this column is the position of the referenced table's key.
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/key-column-usage.md:79]: I'm not sure if this line and the followings are right. do we have foreign keys?

File: docs/v0.7/en/reference/sql/information-schema/columns.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+- `table_schema`: The name of the schema in which the table with the column is located.
+- `table_name`: The name of the table with the column.
+- `column_name`: The name of the column.
+- `data_type`: The type of data in the column.
+- `semantic_type`: The type of the column. It can be one of the following: `TAG`, `FIELD`, or `TIMESTAMP`.
+- `column_default`: The default value of the column. If the explicit default value is `NULL`, or if the column definition does not include the `default` clause, this value is `NULL`.
+- `is_nullable`: Whether the column is nullable. If the column can store null values, this value is `YES`; otherwise, it is `NO`.
+- `column_type`: The data type of the column. It is the same as the `DATA_TYPE` column.
```
Comment (nicecui) [inline docs/v0.7/en/reference/sql/information-schema/columns.md:83]: @killme2008 please confirm whether they are the same

File: docs/v0.7/en/reference/sql/information-schema/greptime-region-peers.md (hunk 4 above)
Comment (killme2008) [reply docs/v0.7/en/reference/sql/information-schema/greptime-region-peers.md:6]: confirm what? It's not available on cloud for security reason.

File: docs/v0.7/en/reference/sql/information-schema/key-column-usage.md (hunk 5 above)
Comment (killme2008) [reply docs/v0.7/en/reference/sql/information-schema/key-column-usage.md:79]: Of course, we don't support it, but these columns must be present for compatibility.

File: docs/v0.7/en/reference/sql/information-schema/columns.md (hunk 6 above)
Comment (killme2008) [reply docs/v0.7/en/reference/sql/information-schema/columns.md:83]: If I wrote it, it must be the same. :D

File: docs/v0.7/en/reference/sql/information-schema/table-constraints.md (hunk 2 above)
Comment (killme2008) [reply docs/v0.7/en/reference/sql/information-schema/table-constraints.md:2]: Yep, so it's `TODO`, we have to support it.

**Review by nicecui — APPROVED** [review]

**Comment by tisonkun** [comment]
I'll review this PR tomorrow.

File: docs/v0.7/zh/reference/sql/information-schema/columns.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
++--------------------------+--------+------+------+---------+---------------+
+24 rows in set (0.00 sec)
+```
+
+创建表 `public.t1` 并查询其在 `COLUMNS` 中的信息：
+
+```sql
+CREATE TABLE public.t1 (h STRING, v FLOAT64, ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX, PRIMARY KEY(h));
+SELECT * FROM COLUMNS WHERE table_schema='public' AND TABLE_NAME='t1'\G
+```
```
Comment (tisonkun) [inline docs/v0.7/zh/reference/sql/information-schema/columns.md:49]:
````suggestion
```sql
CREATE TABLE public.t1 (h STRING, v FLOAT64, ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX, PRIMARY KEY(h));
SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE table_schema='public' AND TABLE_NAME='t1';
```
````

Comment (tisonkun) [reply docs/v0.7/zh/reference/sql/information-schema/columns.md:49]: Doesn't seems necessary.

File: docs/v0.7/en/reference/sql/information-schema/overview.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+| `ROUTINES` | Not implemented. Returns zero rows. |
+| [`SCHEMATA`](./schemata.md) | Provides similar information to `SHOW DATABASES`. |
+| `SCHEMA_PRIVILEGES` | Not implemented. Returns zero rows. |
+| `SESSION_STATUS` | Not implemented. Returns zero rows. |
+| `SESSION_VARIABLES` | Not supported. |
+| `STATISTICS` | Not supported. |
+| [`TABLES`](./tables.md) | Provides a list of tables that the current user has visibility of. Similar to `SHOW TABLES`. |
+| `TABLESPACES` | Not supported. |
+<!-- | [`TABLE_CONSTRAINTS`](./table-constraints.md) | Provides information on primary keys, unique indexes, and foreign keys. | -->
+| `TABLE_PRIVILEGES` | Not implemented. Returns zero rows. |
```
Comment (tisonkun) [inline docs/v0.7/en/reference/sql/information-schema/overview.md:42]: Malformed:

image

**Review by tisonkun — CHANGES_REQUESTED** [review]

**Comment by nicecui** [comment]
@tisonkun fixed, PTAL

**Review by tisonkun — APPROVED** [review]

**Pull request merged**